pkg/apis/noobaa/v1alpha1/noobaa_types.go

```diff
@@ -227,6 +227,11 @@ type NooBaaSpec struct {
 	// BucketLogging sets the configuration for bucket logging
 	// +optional
 	BucketLogging BucketLoggingSpec `json:"bucketLogging,omitempty"`
+
+	// NotificationsPVC (optional) specifies the name of the Persistent Volume Claim (PVC) to be used
+	// for notifications persistent files.
+	// +optional
+	NotificationsPVC *string `json:"notificationsPVC,omitempty"`
```

Comment:
Following a discussion with Amit and Jacky:
* Add `BucketNotificationsSpec` that contains `enable`, optionl `pvc`, array of secrets.
* we will use separate PVCs for bucket logging and bucket notifications.
* Reuse common code between notifications and logging.

Reply:
Changes are the two new commits from this week, LMKWYT